jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimDepositController.java

```diff
@@ -105,6 +109,27 @@ public TimDepositController(OdeProperties odeProperties) {
       ? Boolean.parseBoolean(System.getenv("DATA_SIGNING_ENABLED_SDW"))
       : true;
 
+      // start the TIM ingest monitoring service if enabled
+      Boolean timIngestMonitoringEnabled = Boolean.valueOf(odeProperties.getProperty(ConfigEnvironmentVariables.ODE_TIM_INGEST_MONITORING_ENABLED));
+      if (timIngestMonitoringEnabled) {
+         logger.info("TIM ingest monitoring enabled.");
+         
+         ScheduledExecutorService scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
+         // 3600 seconds, or one hour, was determined to be a sane default for the monitoring interval if monitoring is enabled
+         // but there was no interval set in the .env file
+         String interval = odeProperties.getProperty(ConfigEnvironmentVariables.ODE_TIM_INGEST_MONITORING_INTERVAL);
+         // getProperty(name, default) method will not use the default value if the value is set to an empty string, so we are using getProperty(name) 
+         // and then checking if it is null or empty to protect against the case where the value is set to an empty string in the .env file so that we can
+         // use Long.valueOf() without risk of a NumberFormatException. 
+         if (interval == null || interval.isEmpty()) {
+            interval = "3600";
+         }
+         Long monitoringInterval = Long.valueOf(interval);
```

Comment:
Should we wrap this setup in a try/catch in case someone enters a non-numerical value?

Reply:
It couldn't hurt to add that. I'll make the change and push it up soon

Reply:
I migrated to using a try-catch to protect from non-numerical input in https://github.com/CDOT-CV/jpo-ode/pull/106/commits/ba5b641336c744ffe6160dcfa14522bdad078285